### DISCUSSION TITLE
Unable to authenticate the request" err="[invalid bearer token, Token has expired.]"

### DISCUSSION THREAD

**Discussion by MagicStarTrace** [k3s-io/k3s#7202] [Q&A]
OS Version: ubuntu 18.04.5

k3s version v1.21.4+k3s1

error log:

controller.go:116] loading OpenAPI spec for "v1beta1.metrics.k8s.io" failed with: failed to retrieve openAPI spec, http error: ResponseCode: 500, Body: Internal Server Error: "/openapi/v2": Unauthorized

3 16:12:47 Ks-Kubernetes-Node01 k3s[911]: E0403 16:12:47.424793     911 authentication.go:63] "Unable to authenticate the request" err="[invalid bearer token, Token has expired.]"

Why does this happen? Restarting k3s service doesn't work either

Thank You!

**Comment by brandond** [comment]
Something is running with a stale token. In this case I suspect it's the metrics-server pod. You might try deleting it and allowing it to be re-created.
